ee/ucp/interlock/usage/service-clusters.md

```diff
@@ -4,205 +4,296 @@ description: Learn how to route traffic to different proxies using a service clu
 keywords: ucp, interlock, load balancing, routing
 ---
 
-## Configure Proxy Services
-With the node labels, you can re-configure the Interlock Proxy services to be constrained to the
-workers for each region. For example, from a manager, run the following commands to pin the proxy services to the ingress workers:
+Reconfiguring Interlock's proxy can take a long time (1-2 seconds) per overlay
```

Comment:
Suggest "...can take between 1-2 seconds per overlay...". The phrase "a long time" is ambiguous.